src/sentry/static/sentry/app/stores/streamTagStore.jsx

```diff
@@ -55,6 +55,30 @@ const StreamTagStore = Reflux.createStore({
         values: getMemberListStoreUsernames(),
         predefined: true,
       },
+      lastSeen: {
+        key: 'lastSeen',
+        name: 'Last Seen',
+        values: ['-1h', '+1d', '-1w'],
+        predefined: true,
+      },
+      firstSeen: {
+        key: 'firstSeen',
+        name: 'First Seen',
+        values: ['-1h', '+1d', '-1w'],
```

Comment:
How do `+` values work for these? Wouldn't that be saying "show me things that were first or last seen tomorrow" or am I misunderstanding?

Reply:
I'm pretty sure +1h would be anything seen one hour or more ago vs. -1h would be anything seen in the last hour

Reply:
Ah, that makes much more sense, though I do wonder why it doesn't use the same `<, >, =` as the timestamp syntax does. Nice to have this stuff surfaced, regardless.

Reply:
was maybe going to link https://docs.sentry.io/learn/search/ in der somewhere but didn't know best way to do that

Reply:
It would be nice to have more descriptive labels for these as it doesn't seem intuitive.